setup.py

```diff
@@ -22,10 +26,11 @@
     long_description_content_type="text/markdown",
     url="https://github.com/nabsabraham/pandas-erd",
     packages=setuptools.find_packages(),
-    package_data={'pandaserd': ['VERSION']},
+    package_data={"pandaserd": ["VERSION"]},
     include_package_data=True,
-    python_requires='>=3.6',
-    install_requires = [
-            'pandas'
-      ]
+    python_requires=">=3.6",
```

Comment:
Not sure how this works, but given that [pandas](https://pypi.org/project/pandas/) requires python >=3.8 already, perhaps best to write that here as well? Or does this refer to the python version this "bare" package (without dependencies) would require?